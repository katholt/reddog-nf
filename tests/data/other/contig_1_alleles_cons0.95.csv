Position,Reference,isolate_2,isolate_23,isolate_18,isolate_1,isolate_37,isolate_32,isolate_22,isolate_20,isolate_30,isolate_29,isolate_28,isolate_25,isolate_19,isolate_27,isolate_21,isolate_33,isolate_26,isolate_35,isolate_36,isolate_34,isolate_31,isolate_24
1158,G,A,G,G,A,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
1644,A,A,A,A,G,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A
2081,G,G,G,G,T,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
2086,T,T,T,T,A,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
2092,A,A,A,A,T,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A
4807,G,G,G,G,A,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
4818,T,T,T,T,C,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
4820,T,T,T,G,T,T,T,T,T,T,T,T,T,-,T,T,T,T,T,T,T,T,T
6268,G,G,G,G,A,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
6473,A,C,A,A,C,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A
6484,T,T,T,T,A,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
6485,G,G,G,G,C,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
6506,C,C,C,C,A,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C
6525,T,G,T,T,A,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
8001,G,G,C,C,G,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C,C
10954,T,G,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
10955,T,T,T,T,G,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
12053,G,G,G,G,C,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
13373,G,G,G,G,A,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
13377,T,C,T,T,G,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T,T
13392,A,A,A,A,G,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A,A
14636,G,G,G,G,T,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G,G
