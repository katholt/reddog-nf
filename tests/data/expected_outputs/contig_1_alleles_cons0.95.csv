../other/contig_1_alleles_cons0.95.csv